### ISSUE TITLE
fix gasEstimateStep 

### ISSUE THREAD

**Issue by olehnikolaiev** [skalenetwork/skaled#1637]
run `gasEstimateStep` with `21k gas` if `txn.gasUsed` is less then `21k`

**Steps to reproduce** 
1. deploy contract on chain 
```
// SPDX-License-Identifier: None
pragma solidity ^0.6.0;

contract TestEstimateGas {
    uint256[256] number;
    uint256 counter = 0;

    function store(uint256 x) public {
        number[counter] = x;
        counter += 1;
    }

    function clear(uint256 pos) public {
        number[pos] = 0;
    }
}
```
2. Invoke ``store`` function 
3. send ``eth_estimateGas`` call to skale chain 

**Actual** 
skaled returns "0x0" 

**Expected**
 skaled should return ``0x675f`` or similar positive value larger than 21000

**Comment by DmytroNazarenko** [comment]
skaled: 3.17.0-beta.9

**Comment by oleksandrSydorenkoJ** [comment]
Verified on Regression network 
Version with fix 3.17.0-beta.12


3.16.1 

```
 curl -X POST --data '{"jsonrpc":"2.0","method":"eth_estimateGas","params":[{"from":"0x71cBE3fedE33905d4D1Bf2Bd51f9d4A62375e659", "to":"0x5BD598660151483f5e9A25eD1a28B3a5A4a9fc70", "data":"0xc0fe1af80000000000000000000000000000000000000000000000000000000000000000"}], "id": 1 }' http://1.1.1.1:1111
{"id":1,"jsonrpc":"2.0","result":"0x0"}

```

3.17.0-beta.12
```

 curl -X POST --data '{"jsonrpc":"2.0","method":"eth_estimateGas","params":[{"from":"0xb04a7f2Cd74Eb624Df4BD8bed8f7034B5Aed89C0", "to":"0x8057Ed7F06585F491Fd3BbC18B796759d74F6B7F", "data":"0xc0fe1af80000000000000000000000000000000000000000000000000000000000000000"}], "id": 1 }' http://2.2.2.2:1111
{"id":1,"jsonrpc":"2.0","result":"0x675f"}
```
